rasa/core/processor.py

```diff
@@ -715,11 +718,9 @@ async def parse_message(
                 message, tracker, only_output_properties
             )
 
-        logger.debug(
-            "Received user message '{}' with intent '{}' "
-            "and entities '{}'".format(
-                parse_data["text"], parse_data["intent"], parse_data["entities"]
-            )
+        structlogger.debug(
+            "processor.message.parse",
+            parse_data=parse_data,
```

Comment:
At 2nd thought, maybe we are making the log too long by sharing the entire `parse_data`, let's keep it as it was before with `parse_intent`, parse_text etc.